lib/jekyll/theme_builder.rb

```diff
@@ -56,12 +56,27 @@ def write_file(filename, contents)
 
   def create_directories
     mkdir_p(SCAFFOLD_DIRECTORIES)
+    mkdir_p("_sass/#{name}")
   end
 
   def create_starter_files
     %w(page post default).each do |layout|
       write_file("_layouts/#{layout}.html", template("_layouts/#{layout}.html"))
     end
+
+    %w(footer head header).each do |incl|
```

Comment:
I never liked that choice of naming either. But I kept it on the assumption that:
- Joel must have named them according to corresponding HTML5 elements.
- The community never objected to it till now. (Theme developers must've been copying from them minima gem dir anyway. )

I'll change after other maintainers / captains weigh in on this..